ui/i18n/src/main/res/values/strings.xml

```diff
@@ -169,8 +169,7 @@
     <string name="share_label">Share</string>
     <string name="share_label_with_ellipses">Share…</string>
```

Comment:
```suggestion
     <string name="share_label">Share</string>
```
On my phone, sorry - can't change the string name in other places.

I would remove the ellipses because there is no sub-menu any longer – for Episodes at least, I believe you changed it also the mechanism for Podcasts, right? Plus it's cleaner (also for TalkBack).